src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -160,6 +179,88 @@ private void SetPanelSizes(float panel1Fraction)
         _panel2Size = string.Create(CultureInfo.InvariantCulture, $"{(1 - panel1Fraction):F3}fr");
     }
 
+    public async Task OnPageKeyDownAsync(KeyboardEventArgsWithPressedKeys args)
+    {
+        if (_splitterRef is null || !args.ShiftKey)
+        {
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "t")
+        {
+            await HandleToggleOrientation();
+            return;
+        }
+
+        if (args.Key.ToLower(CultureInfo.InvariantCulture) == "o")
+        {
+            _internalShowDetails = !_internalShowDetails;
+            await InvokeAsync(StateHasChanged);
+            return;
+        }
+
+        var hasChanged = false;
+
+        if (args.Key.ToLower() == "r")
+        {
+            ResetPanelSizes();
+            hasChanged = true;
+        }
+
+        GetPanelSizes(_splitterRef.Panel1Size, _splitterRef.Panel2Size, out var panel1Size, out var panel2Size, out var panel1Fraction);
+
+        if (panel1Size is null || panel2Size is null || panel1Fraction is null)
+        {
+            return;
+        }
+
+        if (args.Key == ">")
```

Comment:
My brain is really struggling with `>` moving the splitter bar to the left and `<` moving it to the right. I understand why it is done that way ("greater than" makes details pane bigger, etc) but between the direction of the "arrow" in the angle bracket and the placement on the keyboard (left button moves splitter bar right, right button moves splitter bar left) my brain is struggling with it. But maybe that's just me.

Reply:
As @leslierichardson95 suggested, changed > and < to + and - respectively. Hopefully that helps?